openy_gated_content.module

```diff
@@ -211,6 +215,9 @@ function openy_gated_content_form_alter(&$form, FormStateInterface $form_state,
         $allowedForms[] = $entity_key . '_' . $bundle . '_form';
         $allowedForms[] = $entity_key . '_' . $bundle . '_edit_form';
         $allowedForms[] = $entity_key . '_' . $bundle . '_add_form';
+        if (in_array($bundle, $events_bundle)) {
```

Comment:
We should remove this check and rely on config to have correct entity types and bundles. It means the patch should just add one more value to the `$allowed_forms` array.